charts/cloudzero-agent/values.schema.json

```diff
@@ -10,15 +10,12 @@
         "pattern": "^[a-zA-Z0-9](?:[a-zA-Z0-9.-]{0,253}[a-zA-Z0-9])?$"
     },
     "cloudAccountId": {
-        "type": [
-            "string",
-            "integer"
-        ]
+        "type": "string"
```

Comment:
I think the only thing with this is that we don't really control what the user puts in and how schema validation interprets that. we want users to be able to `--set cloudZccountId=123456789`. I believe you can force that input to be interpreted as a string, but I think it might just be easier to allow ints and cast to a string internally